
---
bool connection
float64 distance_x
float64 distance_y
float64 distance_z
float64 position_x
float64 position_y
float64 position_z